.php_cs.dist

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$config = new PrestaShop\CodingStandards\CsFixer\Config();
+
+$config
+    ->setUsingCache(true)
+    ->getFinder()
+    ->in(__DIR__)
+    ->exclude('vendor');
+
+return $config;
```

Comment:
What is this file for? Checking the code of this repository? We should enable PrettyCI

Reply:
Yep it check this repository.
Agree to enable PrettyCI!